### PR TITLE
Add text about how to add pattern names

### PR DESCRIPTION
**Pull request by tomschr**
Changes proposed in this pull request:

* Extend section `_adding-and-removing-packages` with how to add patterns
* Taken from the discussion "How to specify package patterns?" on 2020-06-05T10:18

### CODE REVIEW

**Comment by schaefi** [comment]
Thanks Tom, this was a good idea. As I plan to convert the xml description from the former docbook source as described in #1421 I fear this will conflict with the information you added here right now.

My suggestion is to keep this open and merge it into the converted area. I can do that once I'm done with it, ok ?

**Comment by tomschr** [comment]
Thanks Marcus! All good, do whatever you think is useful. :+1: 
I just thought, it would be helpful to not forget it as I consider it useful information.

File: doc/source/working_with_kiwi/xml_description.rst [hunk 1]
```diff
@@ -757,6 +759,30 @@ configured via the following attributes:
 We will describe the different child elements of `packages` in the following
 sections.
 
+If you want to install patterns, use the `namedCollection` element:
```
Comment (Conan-Kudo) [inline doc/source/working_with_kiwi/xml_description.rst:762]: This should be generalized. `namedCollections` is how you install "groups" in RH/Fedora and "patterns" in SUSE, so this should probably be reworded to account for that.

Comment (schaefi) [reply doc/source/working_with_kiwi/xml_description.rst:762]: yes I did that when I added the information to the current xml_description_doc_update branch. Thanks

**Comment by schaefi** [comment]
@tomschr I have added this information in addition to the massive doc update I'm working on in the xml_description_doc_update branch

**Pull request closed without merging**